commands/index.js

```diff
@@ -2,7 +2,7 @@
 
 // Load commands here
 const apiai = require('apiai');
-const app = apiai(process.env.API_AI_TOKEN);
+const api_ai = apiai(process.env.API_AI_TOKEN);
```

Comment:
Choose a case style and stick with it please, make this `const apiAI` to keep with previous case style.